### DISCUSSION TITLE
بخش بحث و گفتگو در مورد این افزونه

### DISCUSSION THREAD

**Discussion by SCR-IR** [SCR-IR/gnome-shamsi-calendar#4] [عمومی]
## 👋 خوش آمدید !
<div dir=rtl>
  اینجا برای بحث، گفتگو و تبادل‌نظر است
<br>
هرچه می‌خواهد دل تنگت نـویس...
</div>

**Comment by mimsadAlef** [comment]
سلام
بهتر نبود قابليت هاي اضافه كردن اذان هاي ديگر رو اضافه مي كرديد

**Reply by SCR-IR** [reply]
<div dir=rtl>
سلام
<br/>
لطفاً بیشتر درمورد منظورتون توضیح بفرمایید. مگه الآن چنین قابلیتی وجود نداره؟
</div>

**Reply by mimsadAlef** [reply]
نه منظور من رو اشتباه فهمیدید
من الان می خوام یه فایل اضافه بکنم باید دونه دونه برای همه اضافه بکنم
یه قابلیت می خوام موذن اضافه بکنم و فایل صوتی رو بهش بدم
و اینکه قابلیت رندوم هم وجود داشته باشه یعنی بشه هر بار یه موذن گفته بشه

**Reply by SCR-IR** [reply]
<div dir=rtl>
پیشنهادات شما خیلی جالب و کاربردی هستند.
<br/>
امّا بیشتر از اینکه ما را به‌سمت ارتقاء این افزونه هدایت کنند، برای ایجاد یک افزونه‌ی جداگانه ترغیب می‌کنند.
<br/>
افزونه‌ای که الآن مورد بحث ماست، تقویم رسمی ایران هست و شلوغ‌کردن بخش اوقات شرعی آن، معمول نیست. چراکه همین الآن هم بخش اوقات شرعی بسیار پیشرفته‌تر از یک تقویم ساده هست و خودمان هم فکر نمی‌کردیم تا اینجا تکمیل شود. فقط با یاری خدا بود!
<br/>
امّا برای امکانات متنوّع و اضافه‌تر، بهتر هست یک نرم‌افزار یا افزونه‌ی مذهبی مخصوص «نماز و تکالیف شرعی» ایجاد شود و امکانات فراوان مرتبط در آن جای بگیرد.
</div>

**Comment by mehrzadtajkarimi** [comment]
سلام متشکرم بابت همه زحماتی که میکشید
تو یک ماه اخیر نرم افزار برای من  از کار افتاد 
و تو ساید بار نمایش داده نمیشه 
دلیلش رو نمیدونم چیه حتی یک بار مجدد فرایند نصب رو انجام دادم
از اوبونتو 22.04 استفاده میکنم

**Reply by SCR-IR** [reply]
سلام
لطفاً آخرین نسخه:
[https://github.com/SCR-IR/gnome-shamsi-calendar/releases](https://github.com/SCR-IR/gnome-shamsi-calendar/releases)
را طبق راهنمایی که در همان صفحه نوشته شده، نصب و آزمایش بفرمایید.

درصورت اجرا نشدن، کد زیر:
```sudo journalctl /usr/bin/gnome-shell | grep 'shamsi-calendar@gnome.scr.ir'```
را درون ترمینال، اجرا و آخرین خطاهای اعلامی را در اینجا یا بخش
Issues
ارسال نمایید.

**Comment by ehsanrs2** [comment]
سلام
تشکر از افزونه خوب شما. به نظر اگر می‌شد که هنگام اذان گفتن صوت اذان را استاپ کنیم هم قابلیت خوبی باشه چون اگر مثلا من در حال کاری باشم که نیاز به صوت باشه تا پایان اذان باید صبر کنم.
